dts/bindings/regulator/cirrus,cp9314.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 Cirrus Logic, Inc.
+# SPDX -License-Identifier: Apache-2.0
+
+description: Cirrus CP9314 Buck Switched Cap DC/DC Converter
+
+compatible: "cirrus,cp9314"
+
+include:
+  - name: i2c-device.yaml
+  - name: regulator.yaml
```

Comment:
shouldn't this filter allowed props?

Reply:
ack. The only regulator we can use right now is `regulator-boot-on`. Some protection properties may be added from `regulator.yaml` down the line.